tests/backstop/scenarios.js

```diff
@@ -1,8 +1,103 @@
+/**
+ * Backstop scenarios.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
 const storybookHost = require( './detect-storybook-host' );
 const rootURL = `${ storybookHost }iframe.html?id=`;
-const storybookStories = require( '../../.storybook/storybook-data' );
+const legacyStorybookScenarios = require( '../../.storybook/storybook-data' );
+const filePaths = require( '../../.storybook/main' );
+const glob = require( 'glob' );
+const fs = require( 'fs' );
+const parser = require( '@babel/parser' );
+const traverse = require( '@babel/traverse' ).default;
+const csf = require( '@componentdriven/csf' );
+const kebabCase = require( 'lodash/kebabCase' );
+
+const newBackstopTests = [];
+
+const storyFiles = glob
+	.sync( filePaths.stories[ 1 ], { cwd: '.storybook' } )
+	// remove relative path from .storybook dir
+	.map( ( path ) => path.slice( 1 ) );
```

Comment:
So here I'm using `cwd` but I still need to remove the relative path `..` (as mentioned on the other comment thread)

Reply:
I've updated it to expand the glob patterns into absolute path patterns, so no need to massage the relative paths anymore 👍